storage-access.bs

```diff
@@ -244,14 +244,16 @@ To <dfn type="abstract-op">determine if a site has storage access</dfn> with [=p
 
 To <dfn type="abstract-op">determine the storage access policy</dfn> for [=partitioned storage key=] |key| with {{Document}} |doc| and {{Promise}} |p|, run these steps:
 
+1. [=Run steps after a timeout=] given |doc|'s {{Window}} object, `"requestStorageAccess"`, `120000`, and the following steps:
```

Comment:
We should add a comment explaining this number. Since it's a number it's not to be wrapped in code.

Similarly, instead of `"requestStorageAccess"` we want "`requestStorageAccess`" (quotes outside code).

And we should probably use `|doc|'s <a>relevant global object</a>` here. (Which we already obtain later, so we should move that step up I suppose.)

Reply:
I will clean up the code quotes and rearrange this.